server/lib/nl/constants.py

```diff
@@ -445,6 +445,13 @@
     EventType.WETBULB: ["WetBulbTemperatureEvent"],
 }
 
+# Given the original place type, this is the order of related place types.
+FALLBACK_CONTAINED_PLACE_TYPES = {
+    "City": ["County", "State", "Country"],
+    "County": ["State", "Country"],
+    "State": ["Country"],
+}
+
```

Comment:
Would it help to use the enums defined for various places (to prevent typo errors): https://github.com/datacommonsorg/website/blob/fa1603e74313bd300a3dd7d547c3a299dd8e63f2/server/lib/nl/detection.py#L95-L113 ?